apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js

```diff
@@ -127,4 +141,118 @@ describe('The JSDebugger component', () => {
 
   });
 
+  describe("Resizing the debug bar", () => {
+    let codeTextbox;
+    beforeEach(() => {
+      codeTextbox = document.createElement('div');
+      codeTextbox.id = 'codeTextbox';
+      document.body.appendChild(codeTextbox);
+    });
+
+    afterEach(() => {
+      document.body.removeChild(codeTextbox);
+    });
+
+    it("mouseup and touchend events on document.body are subscribed to", () => {
+      expect(addEventSpy.withArgs('mouseup')).to.have.been.called;
+      expect(addEventSpy.withArgs('touchend')).to.have.been.called;
+    });
+
+    describe("when the mouse is pressed on the resize bar", () => {
+      beforeEach(() => {
+        resizeBar().simulate('mouseDown');
+      });
+      it("starts listening to mouse move events on the docment body", () => {
+        expect(addEventSpy.withArgs('mousemove')).to.have.been.called;
+        expect(addEventSpy.withArgs('touchmove')).to.have.been.called;
+      });
+
+      describe("when the mouse is moved", () => {
+        const moveTo = (pageY) => {
+          const moveEvent = new CustomEvent('touchmove');
+          moveEvent.pageY = pageY;
+          addEventSpy.withArgs('touchmove').args.forEach(args => args[1](moveEvent));
```

Comment:
I could get events to trigger properly using `$.trigger` or `document.body.dispatchEvent`. So I have to introspect the event handlers and just call them directly. painful and annoying but I guess it works...